gym_maze/maze.py

```diff
@@ -174,3 +183,27 @@ def get_possible_neighbour_cords(pos_x, pos_y) -> tuple:
         nw = (pos_x - 1, pos_y - 1)
 
         return n, ne, e, se, s, sw, w, nw
+
+    def get_goal_state(self, current_x, current_y):
```

Comment:
What is this function returning? I assume some goal perception? How to interpret this?

Reply:
It returns perception of the next goal state.